pkg/anago/anago.go

```diff
@@ -45,6 +45,11 @@ const (
 
 	// releaseNotesJSONFile is the file containing the release notes in json format
 	releaseNotesJSONFile = workspaceDir + "/src/release-notes.json"
+
+	ReleaseDownloadURL = "https://storage.googleapis.com/kubernetes-release/release/"
```

Comment:
`https://storage.googleapis.com/` should be a const somewhere since it is used in various places:
```
> rg -tgo storage.googleapis.com
pkg/release/release.go
254:    urlPrefix := fmt.Sprintf("https://storage.googleapis.com/%s", bucket)

pkg/kubepkg/kubepkg_test.go
525:                    expected: "https://storage.googleapis.com/k8s-artifacts-cni/release/v0.8.6/cni-plugins-linux-amd64-v0.8.6.tgz",

pkg/kubepkg/kubepkg.go
629:    return fmt.Sprintf("https://storage.googleapis.com/k8s-artifacts-cni/release/v%s/cni-plugins-linux-%s-v%s.tgz", version, arch, version), nil

pkg/testgrid/testgrid.go
30:const testgridConfigURL = "https://storage.googleapis.com/k8s-testgrid/config"

cmd/krel/cmd/testgrid.go
206:            testgridJobs[i].GCSLocation = fmt.Sprintf("https://storage.googleapis.com/%s", gcsPath)
```

And then we can construct it as another const, because "kubernetes-release" is already defined in `release.ProductionBucket`.

Reply:
I agree completely, I'll refactor this value in a fallow up. But for now, I removed the `ReleaseDownloadURL` constant from anago since I could not use it in the SBOM.